coralproject/talk
# [CORL-2556] focus on MDE

## What does this PR do?
This PR addresses mouse and keyboard focus state issues found in our audit with the exception of the "Add a message" Markdown Editor.

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?
None.

## Does this PR introduce any new environment variables or feature flags? 
No.

## If any indexes were added, were they added to `INDEXES.md`?
n/a

## How do I test this PR?
1. As an admin mouse user, navigate to stream > Configure > Add a Message or Question
2. Click the button that says "ADD MESSAGE"
3. Note that the editable text field for the Markdown editor has focused and you can begin typing
4. As an admin keyboard user, tab to stream > Configure > Add a Message or Question > "ADD MESSAGE"
5. Use the space or enter button to interact
6. Note that the editable text field for the Markdown editor has focused and you can begin typing
 
 
## How do we deploy this PR?
No special considerations should be required.

## TODO
- [ ] Add an image as a mobile user
- [ ] Add an image as a tablet user
- [ ] Reply button as mobile user: Unsure of feasibility (explicitly disabled on ios)
- [ ] Reply button as a tablet user: Unsure of feasibility (explicitly disabled on ios)
- [ ] Add a gif button as a mobile user
- [ ] Add a gif button as a tablet user
- [ ] Change Email button as a mobile user
- [ ] Change Email button as a tablet user
- [ ] Change username as a mobile user
- [ ] Change username as a tablet user
- [ ] Change password button as a keyboard user
- [ ] Change password button as a mobile user
- [ ] Change password button as a table user
- [x] Add a message button as a mouse user
- [x] Add a message button a sa keyboard only user
- [ ] Add a message button as a mobile user
- [ ] Add a message button as a tablet user
